### DISCUSSION TITLE
小狼毫是否可以控制全盘布局

### DISCUSSION THREAD

**Discussion by yue-dongchen** [rime/home#1434] [Q&A] [answered]
各位好，

我想用[Neo2布局](url)打拼音（或者至少罗马字母模式时使用Neo2布局）。有类似于[Squirrel的全局布局配置吗](https://github.com/rime/squirrel/blob/master/data/squirrel.yaml#L6)？设定系统里已安装的dll布局

所以如果不行，是通过translator实现吗（比如[此Colemak方案的词库](https://github.com/gkovacs/rime-spanish/blob/master/spanish_colemak.dict.yaml)）？
谢谢。

编辑：还是单键[用“punctuator”映射](https://github.com/biopolyhedron/rime-uyghur/blob/b5e208512d902115c105da9cc92d1d27b75fbf4f/uyghur.schema.yaml#L27C12-L27C12)？

**Comment by yue-dongchen** [comment, marked as answer]
嗯。貌似不行。不过我目前改用了Linux配Fcitx，可以内部映射；所以目前没这个需求。

**Reply by fxliang** [reply]
https://github.com/jtroo/kanata
如果不仅仅是小狼毫的需求，可以试试这个做mapping
